azure/appservice-windows/main.tf

```diff
@@ -0,0 +1,89 @@
+resource "azurerm_resource_group" "app" {
+  name     = "rg-${var.project}-${var.environment}"
+  location = local.location
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+    product     = var.product
+  }
+}
+
+resource "azurerm_app_service_plan" "app" {
+  name                = "asp-${var.project}-${var.environment}"
+  location            = azurerm_resource_group.app.location
+  resource_group_name = azurerm_resource_group.app.name
+  kind                = "Windows"
+  reserved            = false
+
+  sku {
+    tier = var.plan
+    size = var.size
+  }
+
+  tags = {
```

Comment:
The tags I believe can be removed, since the tags will be inherited from the RG.

Reply:
done